src/Console/Command/FixCommand.php

```diff
@@ -275,4 +288,23 @@ private function calculateExitStatus($isDryRun, $hasChangedFiles, $hasInvalidErr
 
         return $exitStatus;
     }
+
+    private function getColumns(ConfigurationResolver $resolver)
+    {
+        $columns = $resolver->getColumns();
+        if (null === $columns) {
+            return null;
+        }
+
+        if ('max' === $columns) {
+            // Terminal was introduced in symfony/console 3.2
+            if (!class_exists(Terminal::class)) {
+                return $this->getApplication()->getTerminalDimensions()[0];
+            }
+
+            return (new Terminal())->getWidth();
```

Comment:
ah, great, so we cannot use only old interface as it's deprecated and we cannot use only new interface as it's not backported as compatibility layer...